src/modm/processing/fiber/fiber.hpp.in

```diff
@@ -40,13 +53,20 @@ class Channel;
  * @author Niklas Hauser
  * @ingroup	modm_processing_fiber
  */
+%% if multicore
+template <size_t Core = 0>
+%% endif
```

Comment:
No, I don't like that, it creates a new type that makes it difficult for other code to call the right type.
We would then also need this switch in every other module that uses Fibers.

We should select the right scheduler instance at runtime via the CPUID register and also add another (CPUID defaulted) argument to the constructor to explicitly choose the CPU, but that needs to be properly mutex guarded if you want to insert a fiber into the other core, perhaps via FIFO?